packages/components/src/LightBox/LightBox.css

```diff
@@ -8,7 +8,7 @@
   display: block;
   width: 100%;
   height: 100%;
-  background-color: var(--color-greyBlue);
+  background-color: var(--color-overlay);
   opacity: 0.8;
```

Comment:
I _think_ you need to lose this `opacity` value since the `color-overlay` handles the opacity internally.